styles/layout.css

```diff
@@ -1,229 +1,203 @@
+}
+.header_container header.sticky .img_logo {
+    filter: invert(0%);
+}
+.header_container header .logo {
+    text-transform: uppercase;
+    text-decoration: none;
+    letter-spacing: 0.2rem;
+    color: #fff; 
+}
+.header_container header ul {
+    position: relative;
+    display: flex;
+    align-items: center;
+}
+.header_container header ul li {
+    position: relative;
+    list-style: none;	
+}
+.header_container header ul li a {
+    position: relative;
+    display: inline-block;
+    margin: 0 var(--mar-md);
+    color: #fff;
+    cursor: pointer;
+    text-decoration: none;
+}
+.header_container header ul li a:hover {
+    color: #fff;
+}
+.header_container header.sticky ul li a:hover {
+    color: var(--primary-color);
+}
+.left_auto {
+    margin-left: auto;
+}
+.mobile_menu li {
+    width: 5%;
+}
+/* Aside For Entire Project */
+.aside {
+    padding: var(--pad-xxl) var(--pad-xxl) 0 0;
+    box-shadow: 0px 5px 9px 1px #000000a6;
+    position: relative;
+    grid-column: aside;
+    margin: 5rem 0;
+}
+.container .aside .side-list ul li {
+    display: flex;
+    align-items: center;
+    width: 26rem;
+    margin: 2rem;
+}
+.container .aside .side-list ul li:hover {
+    background-color:var(--primaryColorFaint);
+    border-bottom: 3px solid var(--primary-color);
+}
+.container .aside .side-list ul li svg {
+    margin-right: 4rem;
+    color: var(--primary-color);
+}
+.container .aside .side-list ul li a {
+    text-decoration: none;
+    color: var(--primary-color);
+}
+/* Main Section For Entire Project */
+.main-content {
+    grid-column:main;
+    padding: var(--pad-lg) var(--pad-xxl);
+    margin: var(--mar-infinite) 0;
+}
+/******  Layout classes ********/
+.basic_container {
+    width:100%;
+    border: 1px solid;
+}
+.basic_container iframe {
+    margin: 0 auto;
+    margin: 1rem auto;
+    display: flex;
+}
+.basic_container img {
+    margin: 0 auto;
+    margin: 1rem auto;
+    display: flex;
+}
+.img_container {
+    width: max-content;
+    border: 1px solid;
+}
+/* Modal Container Layout classe Starting Here */
+.modal_cont {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+/* Button Layout container Starting Here */
+.button_cont {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+/* Avatar Starting Here */
+.avt_cont {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: baseline;
+    
+}
+.cursor_ {
+    cursor: pointer;
+}
+/******  Colors ********/
+.secondory-text {
+    color:var(--secondory-text-color)
+}
+.bold {
+    font-weight:800;
+}
+.imp-highlight-text {
+    color:var(--primary-color);
+    border: 1px solid transparent;
+    background: var(--primaryColorFaint);
+}
+.highlightMainText {
+    color:var(--primary-color);
+}
+/* Utility Class system */
+.hide {
+    opacity: 0;
+    pointer-events: none;
+}
+.deco-none {
+    text-decoration: none;
+}
```

Comment:
In general a suggestion, for better readability leave one line after each CSS block

Reply:
for another review one peer member mentioned don't leave spaces so whose advice follow now